sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -478,6 +486,8 @@ export class AssessmentController {
     const institutionLocation = offering?.institutionLocation;
     return {
       applicationId: application.id,
+      applicationStatus: application.applicationStatus,
+      noaApproval: noaApproval,
```

Comment:
Instead of `noaApproval: noaApproval,` it can be simplified as `noaApproval,`.
Please also note the renaming suggested from `noaApproval` to `hasNOAApproval`.

Reply:
Updated